docker-compose/common-services/common-services.yaml

```diff
@@ -12,8 +12,37 @@ services:
       - .:/config
     ports:
       - "8081:8081" # reference-server
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
 
-  kafka_a:
+  observer:
+    image: stellar/anchor-platform:edge
+    build:
+      context: ../../
+      dockerfile: integration-tests/docker-compose-configs/Dockerfile
+    command: "--stellar-observer"
+    environment:
+      - SECRET_SEP10_SIGNING_SEED=SAKXNWVTRVR4SJSHZUDB2CLJXEQHRT62MYQWA2HBB7YBOTCFJJJ55BZF
```

Comment:
Can you change this to 
`secret.sep10.signing_seed`?

Reply:
Done